configs/mainnet.yaml

```diff
@@ -146,9 +146,7 @@ MAX_REQUEST_BLOCKS_DENEB: 128
 MIN_EPOCHS_FOR_BLOB_SIDECARS_REQUESTS: 4096
 # 6 subnets
 BLOB_SIDECAR_SUBNET_COUNT: 6
-# 6 blobs
-MAX_BLOBS_PER_BLOCK: 6
```

Comment:
removing this constant might be more problematic then what we discussed in the other PR https://github.com/ethereum/consensus-specs/pull/4322#issuecomment-2886089239 as it is consensus critical and at least lodestar vc enforces this right now at startup

we can make this more lax now already to avoid potential interop issues

Reply:
it sounds like people are more willing to wear the pain than i'd like so whatever is decided on that other ticket is arguably equally applicable here.